pkgs/by-name/mi/migrate-to-uv/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  python3,
+  fetchFromGitHub,
+  cargo,
+  rustPlatform,
+  rustc,
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "migrate-to-uv";
+  version = "0.2.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "mkniewallner";
+    repo = "migrate-to-uv";
+    rev = version;
+    hash = "sha256-LA2tzTD3e6IPmeYHWKFD+PIsl6hsvfpYDKhN9upttHI=";
+  };
+
+  cargoDeps = rustPlatform.importCargoLock {
+    lockFile = ./Cargo.lock;
+  };
```

```suggestion
  cargoDeps = rustPlatform.fetchCargoVendor {
    inherit src pname version;
    hash = "sha256-aiUCLRHCntJKZGCNdyfFCyRdIP+9Fr8yVzaDVct9Dv8=";
  };
```

Since the `Cargo.lock` provided by upstream seems to be up to date you don't need to add this file here